doajtest/testbook/articles_preservation/user_test_script.yml

```diff
@@ -75,3 +75,27 @@ tests:
     - if the status is 'partially success', show details link will be displayed. When
       clicked on the link details of how many articles successful and how many not
       successful will be displayed
+- title: Upload correctly structured file with multiple journals and it is processed correctly
+  context:
+    role: publisher
+  steps:
+  - step: Ensure that the archive package has been created with correct structure
+  - step: Ensure that the archive package contains more than one article with different journals. 
+          A sample zip file is available for testing purpose. If the articles provided in zip are not available in the elasticsearch data, 
+          the results may not be as expected. In that case unzip the file and update the identifiers.txt file with correct identifier for the articles.
+    resource: /preservation_upload_test_package/multi_journals.zip
```

Comment:
Can we also add an example package resource to the other relevant tests:

https://doaj.github.io/doaj-docs/feature/3424_archiving_project/testbook/index.html#articles_preservation/user_test_script/upload_correctly__structured_file_and_it_is_processed_correctly

https://doaj.github.io/doaj-docs/feature/3424_archiving_project/testbook/index.html#articles_preservation/user_test_script/upload_incorrectly__structured_file

https://doaj.github.io/doaj-docs/feature/3424_archiving_project/testbook/index.html#articles_preservation/user_test_script/upload_correctly__structured_file_with_articles_that_user_do_not_own